app/controllers/search_controller.rb

```diff
@@ -13,10 +13,13 @@ def extract_filters
 
   def log_metrics_event
     if params[:type] && params[:data] then
-      if params[:other_data] && params[:other_data] != "" then
-        metrics_event(params[:type], params[:data], eval(params[:other_data])) 
-      else 
-        metrics_event(params[:type], params[:data])
+      if params[:type] == 'reverb_redirect'
+        if params[:other_data] && params[:other_data] != "" then
+          
+          metrics_event(params[:type], params[:data], eval(params[:other_data])) 
+        else 
```

Comment:
scrub all the data in ```:data```, ```:other_data``` in this "default" block.

Reply:
I will, though I think that's validation too far - metrics are nothing but key/value string pairs.  If we add other metrics handling on the front end, this strict validation could easily become unwieldy.

Reply:
This is a public facing endpoint. If later we wanna use this at all (note all of these will be reverted), we should do the same. The data passed to default ```else``` should always be scrubbed. Plus, we mainly use google analytics for js metrics. Adding this endpoint is for quicker retrieval of metrics from splunk using reports.

Reply:
@mightynimble The data passed to the default else *is* scrubbed - the default else uses the type and data, please look at the conditional.  Both of those have been validated before it hits that else.